bw_add_sshkeys.py

```diff
@@ -7,12 +7,17 @@
 import json
 import logging
 import os
+import sys
 import subprocess
 from typing import Any, Callable, Dict, List, Optional
+from cryptography.hazmat.primitives.serialization import load_ssh_private_key
```

Comment:
Currently, we don't have any external dependencies (besides bitwarden-cli and ssh-agent, of course). Adding an external library as dependency would make this tool harder to install.

We might need to rethink if that's the direction we'd like to go, but for now I'd say we better avoid requiring extra libs.

Reply:
Hi @joaojacome,

Basically I agree with you, a new library is of course always a problem. 
But I still decided to go this way, because at the moment the user has to manually remove the just added keys from the ssh-agent. From the user's point of view, I find this very inconvenient, so I decided to add the "Key Removal" function. Therefore the library is "required".

Additionally, due to your concerns, I tried to read up a bit about this library. Basically this library seems to be relatively well known, has thousands of downloads on pypi.org and a clean release management. I also checked the default package lists of Debian and Manjaro (desktop OS of my choice). In both operating systems this package (python-cryptography) is included in the default lists. So the coverage should be relatively wide.

Maybe you could also start a poll among users of the tool?

But it's your choice, of course. Feel free to reject the PR as is.

Greetings
hasechris :-)

Reply:
Might be a compromise to leverage the ssh-agent timeout, to have simple way to restrict the maximum time a key is active?

We could simply pass a `-t` argument to the subprocess call. That's straight forward, needs no additional libraries, but would prevent keys from staying in memory indefinitely.